pkg/engine/github.go

```diff
@@ -21,9 +21,11 @@ func (e *Engine) ScanGitHub(ctx context.Context, c sources.GithubConfig) error {
 		ScanUsers:                  c.IncludeMembers,
 		IgnoreRepos:                c.ExcludeRepos,
 		IncludeRepos:               c.IncludeRepos,
+		IncludeForks:               c.IncludeForks,
 		IncludeIssueComments:       c.IncludeIssueComments,
 		IncludePullRequestComments: c.IncludePullRequestComments,
 		IncludeGistComments:        c.IncludeGistComments,
+		IncludeWikis:               c.IncludeWikis,
```

Comment:
where is `IncludeWikis` actually checked?

Reply:
Oh dear.

Reply:
An oversight on my part. It should be checked now in `repo.go`.